app/views/order_items/_order_item.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="col-md-3 form_products">
-  <div class="thumbnail ">
+  <div class="thumbnail<%= product.stock ==0 ? ' out-of-stock' : '' %>">
```

Comment:
Right, english.



On 8 Jul 2015 at 14:43:54, benji (notifications@github.com) wrote:

In app/views/order_items/_order_item.html.erb:

> @@ -1,5 +1,5 @@
>  <div class="col-md-3 form_products">
> -  <div class="thumbnail ">
> +  <div class="thumbnail<%= product.stock ==0 ? ' out-of-stock' : '' %>">
.nil? doesn't return true for 0. zero? does.

I don't think you need a ternary operator here. <%= ' out-of-stock' if product.stock.zero? %> should work too.

—
Reply to this email directly or view it on GitHub.